packages/staking/src/governance/manager.test.ts

```diff
@@ -0,0 +1,128 @@
+import {describe, it, expect, afterEach} from '@jest/globals'
+
+import {createGovernanceManager, GovernanceAction} from './manager'
+import {init} from '@emurgo/cross-csl-nodejs'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import {initApi} from './api'
+import {axiosClient} from './config'
+
+describe('createGovernanceManager', () => {
+  const cardano = init('global')
+  const networkId = 1
+  const options = {
+    networkId,
+    cardano,
+    storage: AsyncStorage,
+    api: initApi({networkId, client: axiosClient}),
+  }
+
+  afterEach(async () => {
+    await options.storage.clear()
+  })
+
+  it('should return a manager', () => {
+    expect(createGovernanceManager(options)).toBeDefined()
+  })
+
+  describe('validateDRepID', () => {
+    const governanceManager = createGovernanceManager(options)
+
+    it('should throw an error when for a DRep id that has wrong length', async () => {
+      const invalidId = 'abc'
+      const errorMessage = 'invalid DRep ID length, must be 56 characters'
```

Comment:
so it's just the hash no bech32 ? (I'd say the client api should receive both and we transform before consuming the api)